index.js

```diff
@@ -63,6 +71,15 @@ function createFromElement(el) {
 function getElementProperties(el) {
   var obj = {}
 
+  // IE8 data attributes
+  if (dataAttrs.length > 0) {
+    dataAttrs.forEach(function(dataAttr) {
+      if (el.getAttribute(dataAttr)) {
+        obj[dataAttr] = el.getAttribute(dataAttr);
```

Comment:
Is this, how vdom wants data-attribs? Does this work? (I assumed it only accepted DOM properties, and  setting the name of an attrib as a property doesn't seem to make sense with that assumption in mind...)